deploy/health-check.sh

```
@@ -0,0 +1,619 @@
+    local env_file="$PROJECT_ROOT/backend/.env"
+
+    if [[ ! -f "$env_file" ]]; then
+        ISSUES_FOUND+=("Environment file not found: $env_file")
+        return 1
+    fi
+
+    # Check for required environment variables
+    local required_vars=("OPENAI_API_KEY" "ELEVENLABS_API_KEY" "NODE_ENV")
+    local missing_vars=()
+
+    source "$env_file" 2>/dev/null || {
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)